.templates/perl/default.mk

```diff
@@ -42,28 +68,31 @@ else
 endif
 .PHONY: update-version
 
-.cpanfile_dependencies: cpanfile
-	PERL5LIB=${PERL5LIB} cpanm --notest --local-lib ./perl5 --installdeps .
-	touch $@
-
-predistribution: dist-clean test CHANGELOG.md
-# --notest to keep the number of dependencies low: it doesn't install the
-# testing dependencies of the dependencies.
-	cpanm --notest --local-lib ./perl5 --installdeps --with-develop .
-	cpanm --notest --local-lib ./perl5 'Dist::Zilla'
-	PERL5LIB=${PERL5LIB} PATH=$$PATH:${PERL5PATH} dzil authordeps --missing | cpanm --notest --local-lib ./perl5
-	PERL5LIB=${PERL5LIB} PATH=$$PATH:${PERL5PATH} dzil clean
-	@(git status --porcelain 2>/dev/null | grep "^??" | perl -ne\
-	    'die "The `release` target includes all files in the working directory. Please remove [$$_], or add it to .gitignore if it should be included\n" if s!.+ perl/(.+?)\n!$$1!')
-.PHONY: predistribution
 
-pre-release: update-version
-.PHONY: pre-release
+publish: dist
+ifeq ($(IS_TESTDATA),-testdata)
+	# no-op
+else
+ifneq (,$(DISTINI))
+	cpan-upload $(DIST)
```

Comment:
That requires using the new version of the build image?

If so, maybe we should use that new version as part of that PR?

Reply:
(FYI: I've just released a new version of the cucumber build image: 0.13.0)